### ISSUE TITLE
Make line numbers in console output clickable.

### ISSUE THREAD

**Issue by pgr-a** [TeXworks/texworks#935]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I get an error message in console output like

! Undefined control sequence.
l.6414 ...u\rest{\partial M}\right

and then i have to start scrolling. 

**Describe the solution you'd like**
I'd like to click on that number resulting in the editor window jumping to that line.

I realize that it's hard to automatically find out which number is a line number, which file it refers to or if the message is an error message. But none of this is necessary.

My (I think simple to implement) suggestion fixing 95% of the problem is this: After a mouse click into the console output, see if a number is hit, see if the current document has that as a line number, and jump there. Wrong document? My fault. Not a line number? My fault.

My improved suggestion (fixing the remaining 5%): additionally (not instead!) have a right click on a number list the open files (optionally: and TEX root) (optionally: if they have that line number) and allow me to jump to that line number in that file. 

This would have saved me hundreds of minutes in the past.

Thanks for your consideration.

**Comment by stloeffler** [comment]
I agree that this would be a nice enhancement. However, in the meantime, are you using the "Errors, warnings, badboxes" hook script that comes bundled with TeXworks? It should actually parse the console output and provide clickable line numbers (among other things).

**Comment by pgr-a** [comment]
this hook script has a checkmark, apparently by default. i still cant click on any numbers in the output pane. what else is necessary for this to work?
